build/scripts/contiv-vol-run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+# cleanup the older container
+docker rm -f "$1" &>/dev/null || :
+
+## test for shared mount capability
+if ! grep "MountFlags" /lib/systemd/system/docker.service | grep shared &>/dev/null
+then
+  echo 1>&2 "Docker cannot run volplugin in its current state."
+  echo 1>&2 "You must change docker's systemd MountFlags to equal 'shared'"
```

Comment:
in summary, the mounts will not propagate to the host bind-mount unless the shared flag is set on the parent mount point.